### PR TITLE
WEB-8825 get list of apis

### PR DESCRIPTION
**Pull request by TonyXuFX**

### CODE REVIEW

File: Api/Controllers/PartnerApiController.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      IStorageSettings _storageSettings;
+
+      public PartnerApiController(IStorageSettings storageSettings)
+      {
+         _storageSettings = storageSettings ?? throw new ArgumentNullException(nameof(storageSettings));
+      }
+
+      [HttpGet]
+      [ProducesResponseType(200)]
+      [ProducesResponseType(404)]
```
Comment (eozcelik) [inline Api/Controllers/PartnerApiController.cs:34]: we dont return `NotFound()`

File: Api/Controllers/PartnerApiController.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      public async Task<ActionResult<IEnumerable<PartnerApi>>> GetApiList()
+      {
+         var partnerApiTable = new Storage.Services.Repository<PartnerApi>(_storageSettings);
```
Comment (eozcelik) [inline Api/Controllers/PartnerApiController.cs:37]: why not inject IRepositroy<PartnerApi>

Comment (TonyXuFX) [reply Api/Controllers/PartnerApiController.cs:37]: Oh I didn't realize that was a thing. I was looking around and just saw code like this in SearchSupportController

File: Api/Controllers/PartnerApiController.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+   /// <summary>
+   /// Handle notifications
+   /// </summary>
+   [Produces("application/json")]
+   [Route("api/[controller]")]
+   [ApiController]
+   [Authorize]
+   public class PartnerApiController : ControllerBase
+   {
+      IRepository<PartnerApi> _repository;
```
Comment (eozcelik) [inline Api/Controllers/PartnerApiController.cs:20]: `private readonly` qualifier would be good here

**Pull request merged**